pkgs/top-level/all-packages.nix

```diff
@@ -9917,6 +9917,11 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) IOKit;
   };
 
+  pcscliteWithPolkit = pcsclite.override {
+    pname = "pcsclite-with-polkit";
+    polkitSupport = true;
+  };
```

Comment:
```suggestion
  pcscliteWithPolkit = pcsclite.override { polkitSupport = true; };
```

Reply:
I think it's better to do it from all-packages.nix so one can easily see the name has changed.